protocol/Overview.md

```diff
@@ -0,0 +1,58 @@
+# Overview
+
+Bitcoin Cash is a decentralized cryptocurrency with a public distributed ledger. Transactions are validated and transmitted over the Bitcoin Cash peer-to-peer (P2P) network, and stored in the [blockchain](protocol\blockchain). Bitcoin Cash utilizes the Proof-of-work timestamping scheme with SHA-256 hash function, and targets a 10-minute block time. It also utilized the secp256k1 parameters with ECDSA and Schnorr algorithm for digital signatures. The ticker symbol for Bitcoin Cash is BCH.<br>
```

Comment:
> It also utilized the secp256k1

This indicates that it no longer does, which I believe is not the case. Should probably be "utilizes"